MH4U Dex/QuestDetailsTableViewController.m

```diff
@@ -46,20 +49,40 @@ - (void)viewDidLoad
                                @"Quest Fee",
                                @"Zenny Reward",
                                @"HRP Reward",
+                               @"Quest Region",
                                ];
-    self.questDetailsDict = @{
-                               self.questDetailLabels[0]: self.quest.caravan,
-                               self.questDetailLabels[1]: self.quest.stars,
-                               self.questDetailLabels[2]: self.quest.objective,
-                               self.questDetailLabels[3]: self.quest.subObjective,
-                               self.questDetailLabels[4]: self.quest.type,
-                               self.questDetailLabels[5]: self.quest.keyQuest,
-                               self.questDetailLabels[6]: self.quest.urgent,
-                               self.questDetailLabels[7]: self.quest.danger,
-                               self.questDetailLabels[8]: self.quest.fee,
-                               self.questDetailLabels[9]: self.quest.reward,
-                               self.questDetailLabels[10]: self.quest.hrp,
-                               };
+    //TODO: Can this be organized better?
```

Comment:
My thought is to just create the details dict as a mutable dictionary and if there's not a region, replace the value `self.questDetailLabels[11]` since that appears to be the only difference between the two.